src/app/title/page.tsx

```diff
@@ -95,22 +107,45 @@ function Buttons() {
 	);
 }
 
-function DependencyTab() {
+interface DependencyTabProps {
+	titles: TitleDto[];
+	relevance: RelationRelevance;
+}
+
+function DependencyTab({ titles, relevance }: DependencyTabProps) {
 	return (
 		<DependencyTabStyle>
-			<p>must</p>
+			{titles.map((title) => (
+				<MiniPoster
+					key={title.id}
+					src={title.smallPosterUrl}
+					alt={title.name}
+				/>
+			))}
+			<p>{relevance}</p>
 		</DependencyTabStyle>
 	);
 }
 
 const DependencyOrder: React.FC<DependencyOrderProps> = ({ title }) => {
+	const relations = title.relations || [];
+	const must = relations
+		.filter((relation) => relation.relevance === "Must")
+		.map((relation) => relation.title);
+	const should = relations
+		.filter((relation) => relation.relevance === "Should")
+		.map((relation) => relation.title);
+	const could = relations
+		.filter((relation) => relation.relevance === "Could")
+		.map((relation) => relation.title);
```

Comment:
This can be simplified to:

```ts
	const relationsMap: Record<RelationRelevance, TitleDto[]> = {
		Must: [],
		Should: [],
		Could: [],
	};

	for (const relation of relations) {
		relationsMap[relation.relevance].push(relation.title);
	}
```